src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -158,26 +154,23 @@ export default {
         this.observeForChanges();
         this.conditionManager = new ConditionManager(this.domainObject, this.openmct);
         this.conditionManager.on('conditionSetResultUpdated', this.handleConditionSetResultUpdated);
-        this.updateDefaultCondition();
+        this.conditionManager.on('noTelemetryObjects', this.handleNoTelemetryObjects);
         this.stalenessSubscription = {};
     },
     methods: {
         handleConditionSetResultUpdated(data) {
             this.currentConditionId = data.conditionId;
             this.$emit('conditionSetResultUpdated', data);
         },
+        handleNoTelemetryObjects(data) {
```

Comment:
handler functions should be named for what they do. In this case `emitNoTelemetryObjectEvent` or something.